source/parsed_finite_element.cc

```diff
@@ -65,7 +65,8 @@ template <int dim, int spacedim>
 FiniteElement<dim,spacedim> *
 ParsedFiniteElement<dim, spacedim>::operator()() const
 {
-  return FETools::get_fe_by_name<dim,spacedim>(fe_name);
+  // get_fe_by_name returns now a unique_ptr
+  return FETools::get_fe_by_name<dim,spacedim>(fe_name).get();
```

Comment:
The correct approach is the other way around. Make this function return a unique ptr.